tests/test_cli.py

```diff
@@ -1049,6 +1056,51 @@ def test_check_storage_url() -> None:
         optuna.cli._check_storage_url(None)
 
 
+@pytest.mark.skipif(platform.system() == "Windows", reason="Skip on Windows")
```

Comment:
These tests issues following errors on Windows.

> FAILED tests/test_cli.py::test_get_storage_without_storage_class - sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
> (Background on this error at: https://sqlalche.me/e/20/e3q8)
> FAILED tests/test_cli.py::test_get_storage_with_storage_class - sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
> (Background on this error at: https://sqlalche.me/e/20/e3q8)
> = 2 failed, 4103 passed, 88 skipped, 1003 deselected, 164 warnings in 1067.88s (0:17:47) =
> Error in atexit._run_exitfuncs:
> https://github.com/optuna/optuna/actions/runs/5020758022/jobs/9002514278?pr=4696

I checked `tests/storage_tests/rdb_tests/test_storage.py` to resolve above errors, however tests are skipped on Windows as follows. So let me use `pytest.mark.skipif` decorator here.

https://github.com/optuna/optuna/pull/4457/files#diff-42166922cf20058a0456b29a4eccb65307689d7aefed170b335539b5e3ed11eb